rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java

```diff
@@ -127,6 +130,8 @@ public static Menu createContextMenuFromCompactView(RMController controller, Obj
                 lockItem.setEnabled(true);
                 unlockItem.setEnabled(false);
             }
+        } else {
+            copyItem.setEnabled(false);
```

Comment:
You don't need to set this to true anywhere?

Reply:
Normally no, it's enabled by default, but you're right, to be consistent with the other items I'll enable it back in the other case.